main/EDInitMod.F90

```diff
@@ -185,66 +188,61 @@ subroutine set_site_properties( nsites, sites)
     !
     ! !LOCAL VARIABLES:
     integer  :: s
-    real(r8) :: leafon
-    real(r8) :: leafoff
-    logical  :: stat
-    real(r8) :: NCD
+    integer  :: cstat      ! cold status phenology flag
     real(r8) :: GDD
-    logical  :: dstat
+    integer  :: dstat      ! drought status phenology flag
     real(r8) :: acc_NI
-    real(r8) :: watermem
-    integer  :: dleafoff
-    integer  :: dleafon
+    real(r8) :: watermem 
+    real(r8) :: cleafon    ! DOY for cold-decid leaf-on, initial guess
```

Comment:
Why are the `cleafoff` and `cleafon` values set as `real` whereas the `dleaf` states stayed integer?  It looks like based on the `cleafondate` there will be a conversion to integer down the line, correct?

Reply:
correct, so the local is defined as the wrong type